### PR TITLE
Handled API keys in prod and special characters redirect urls

### PR DESCRIPTION
**Pull request by jindaln**

### CODE REVIEW

File: src/main/java/com/google/sps/data/DatastoreManager.java [hunk 1]
```diff
@@ -46,8 +46,8 @@ public static long insertCommentInDatastore(String name, String message) {
      * @param title of representative  
      * @return ID of entity inserted into datastore
      */ 
-    public static long insertRepresentativeInDatastore(String name, String title, 
-    String username, String password) {
+    public static long insertRepresentativeInDatastore(
+    String name, String title, String username, String password) {
```
Comment (epoms) [inline src/main/java/com/google/sps/data/DatastoreManager.java:50]: should be indented.

File: src/main/java/com/google/sps/data/DatastoreManager.java [hunk 2]
```diff
@@ -165,8 +165,8 @@ public static Representative queryForRepresentativeObjectWithName(String repName
      * @param repPassword password of the representative to search datastore for
      * @return the representative's name as string, or null if it was not found in datastore 
      */ 
-    public static String queryForRepresentativeNameWithLogin(String repUsername, 
-    String repPassword) throws EntityNotFoundException{
+    public static String queryForRepresentativeNameWithLogin(
+    String repUsername, String repPassword) throws EntityNotFoundException{
```
Comment (epoms) [inline src/main/java/com/google/sps/data/DatastoreManager.java:169]: same

File: src/main/java/com/google/sps/servlets/RepresentativeListServlet.java [hunk 3]
```diff
@@ -35,7 +34,7 @@
     private final String API_KEY;
 
     public RepresentativeListServlet() {
-        API_KEY = Dotenv.load().get(Constants.CIVIC_API_KEY);
+        API_KEY = System.getenv(Constants.CIVIC_API_KEY);
```
Comment (v1shanker) [inline src/main/java/com/google/sps/servlets/RepresentativeListServlet.java:37]: API_KEY is not a constant. it should be apiKey

**Comment by v1shanker** [comment]
please fix and merge soon. prioritize merging approved PR's so others can develop off of these chanages.

**Pull request merged**